site/en/guide/migrate.ipynb

```diff
@@ -121,16 +119,16 @@
         "id": "1y-W0Mz_zB6Y"
       },
       "source": [
-        "* *Eager execution, `v1.enable_eager_execution()`* : Any code that implicitly uses a `tf.Graph` will fail. Be sure to wrap this code in a `with tf.Graph().as_default()` context. \n",
+        "- *Eager execution, `v1.enable_eager_execution()`* : Any code that implicitly uses a `tf.Graph` will fail. Be sure to wrap this code in a `with tf.Graph().as_default()` context. \n",
         "    \n",
-        "* *Resource variables, `v1.enable_resource_variables()`*: Some code may depends on non-deterministic behaviors enabled by TF reference variables. \n",
+        "- *Resource variables, `v1.enable_resource_variables()`*: Some code may depends on non-deterministic behaviors enabled by TF reference variables. \n",
         "Resource variables are locked while being written to, and so provide more intuitive consistency guarantees.\n",
         "\n",
-        "  * This may change behavior in edge cases.\n",
-        "  * This may create extra copies and can have higher memory usage.\n",
-        "  * This can be disabled by passing `use_resource=False` to the `tf.Variable` constructor.\n",
+        "  - This may change behavior in edge cases.\n",
+        "  - This may create extra copies and can have higher memory usage.\n",
+        "  - This can be disabled by passing `use_resource=False` to the `tf.Variable` constructor.\n",
         "\n",
-        "* *Tensor shapes, `v1.enable_v2_tensorshape()`*: TF 2.0 simplifies the behavior of tensor shapes. Instead of `t.shape[0].value` you can say `t.shape[0]`. These changes should be small, and it makes sense to fix them right away. See [TensorShape](#tensorshape) for examples.\n",
+        "- *Tensor shapes, `v1.enable_v2_tensorshape()`*: TF 2.0 simplifies the behavior of tensor shapes. Instead of `t.shape[0].value` you can say `t.shape[0]`. These changes should be small, and it makes sense to fix them right away. See [TensorShape](#tensorshape) for examples.\n",
```

Comment:
```suggestion
        "- *Tensor shapes, `v1.enable_v2_tensorshape()`*: TF 2.0 simplifies the behavior of tensor shapes. Instead of `t.shape[0].value` you can say `t.shape[0]`. These changes should be small, and it makes sense to fix them right away. Refer to the [TensorShape](#tensorshape) section for examples.\n",
```